wgengine/monitor/monitor.go

```diff
@@ -232,9 +258,18 @@ func (m *Mon) debounce() {
 			m.logf("interfaces.State: %v", err)
```

Comment:
Do you want to check for significant wall time changes when reading interface state fails?

Reply:
Maybe? I dunno.

I can add that in the next commit where I plumb in wake events, as I'll then have a method for it to call.